br/pkg/task/stream.go

```diff
@@ -1125,6 +1128,17 @@ func restoreStream(
 	if err != nil {
 		return errors.Trace(err)
 	}
+	schemasReplace.AfterTableRewritten = func(deleted bool, tableInfo *model.TableInfo) {
```

Comment:
Can we put the function `AfterTableRewritten()` into client.InitSchemasReplaceForDDL() ?

Reply:
Yep, however we must pass the `tiflashRecorder` into `InitSchemasReplaceForDDL`. So I think leave it outside would be fine?🤔

Reply:
OK